plugins/woocommerce/src/Internal/EmailEditor/Integration.php

```diff
@@ -108,7 +108,12 @@ public function add_email_post_type( array $post_types ): array {
 					'search_items'  => __( 'Search Woo Emails', 'woocommerce' ),
 				),
 				'rewrite'  => array( 'slug' => self::EMAIL_POST_TYPE ),
-				'supports' => array( 'title', 'editor' ),
+				'supports' => array(
+					'title',
+					'editor' => array(
+						'default-mode' => 'template-locked',
+					),
+				),
```

Comment:
Hi @lysyjan 
I remembered it would be best to move this fix to the Core Editor PHP package [here](https://github.com/woocommerce/woocommerce/blob/trunk/packages/php/email-editor/src/Engine/class-email-editor.php#L173), and we can remove the supports parameter from this array.

This ensures other extenders receive the update as well when we publish the packages.

Reply:
Hey @triple0t!
This is a good point, but I am afraid that moving the fix is not possible. We also need to add the default mode to the default values, but because there is array_merge, the value from the Integrator has a higher priority. I am going to add a new commit.